### PR TITLE
Cleaned up code a little bit

### PR DESCRIPTION
**Pull request by bhavyaSA**

### CODE REVIEW

File: myscript.js [hunk 1]
```diff
@@ -266,36 +312,24 @@ function clearTwitterPhotos() {
...
-            chrome.storage.sync.set({ toggleTwitterPhotos: toggleTwitterPhotos })
+            twitterUpdater('photos',true)
         }
 
         if (request.toggleTwitterNames) {
-            toggleTwitterNames = !toggleTwitterNames
-            clearTwitterNames();
-            chrome.storage.sync.set({ toggleTwitterNames: toggleTwitterNames })
+            twitterUpdater('names',true)
         }
```
Comment (hyperwidget) [inline myscript.js]: Consider doing this in a switch/case?

Comment (hyperwidget) [reply myscript.js]: actually ignore that for now; disregard

File: myscript.js [hunk 2]
```diff
@@ -1,48 +1,91 @@
...
+    }
+}());
+
+var angellistUpdater = (function () {
+    var toggle = {}
+    toggle['photos'] = [false,clearAlPhotos,TOGGLE_ANGELLIST_PHOTOS]
+    toggle['names'] = [false,clearAlNames,TOGGLE_ANGELLIST_NAMES]
+
+    return function(type,isSet = false,val) {
+        if (val != undefined) {
```
Comment (hyperwidget) [inline myscript.js]: Am I wrong in thinking all these returned functions look the same?

Comment (bhavyaSA) [reply myscript.js]: yea lol they do, can definitely be made more modular

**Pull request merged**